solutions/standard/variables.tf

```diff
@@ -16,7 +16,8 @@ variable "use_existing_resource_group" {
 
 variable "resource_group_name" {
   type        = string
-  description = "The name of a new or an existing resource group in which to provision Secrets Manager resources to. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of a new or an existing resource group in which to provision Secrets Manager resources to. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'. Optional if existing_secrets_manager_crn is set."
```

Comment:
shouldn't be here if `existing_secrets_manager_crn` is **not** set?  
according to `count = var.existing_secrets_manager_crn == null ? 1 : 0` we do not  create resource group if `existing_secrets_manager_crn` is set